src/harness/testcases/configs/test_WINNF_FT_S_SCS_20/default.config

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this used anywhere?

Reply:
This   file has been removed as the associated test case has been removed.